qa/mc_sc_forging1.py

```diff
@@ -208,5 +209,17 @@ def run_test(self):
         check_subommer(scblock_id2, scblock_id1, [mcblock_hash1], scblock_id6, sc_node1)
 
 
+        # Check SC node forging status
+        # Auto forging is disabled.
+        is_forging_enabled = sc_node1.block_forgingInfo()["result"]["forgingEnabled"]
+        assert_equal(False, is_forging_enabled, "Automatic forging expected to be disabled.")
+        # Enable forging
+        if "result" not in sc_node1.block_startForging():
+            fail("Was not able to start auto forging.")
+        # Check the new status
+        is_forging_enabled = sc_node1.block_forgingInfo()["result"]["forgingEnabled"]
+        assert_equal(True, is_forging_enabled, "Automatic forging expected to be disabled.")
```

Comment:
I assume that in this case message should tell that forging is expected to be enabled.